recipes-kernel/images/esp-qcom-image.bb

```diff
@@ -1,14 +1,20 @@
 DESCRIPTION = "EFI System Partition Image to boot Qualcomm boards"
 
 PACKAGE_INSTALL = " \
-    linux-qcom-uki \
```

Comment:
I also noticed that in our current UKI (we build with linux-qcom-uki) we do not set an initrd, while I think there is one by default with the upstream uki class. It's a boot behavior change that we need to be aware of. We might want an initrd, but we should not make this change without an explicit commit/discussion.

Reply:
Initrd or initramfs? They are subtly different in how the / mountpoint is pivoted or not.

Reply:
we use none of them right now.

Reply:
We might want to use initrd with meta-qcom-distro, but here I don't think it is needed / used.